dev/deploy.rst

```diff
@@ -36,12 +36,13 @@ It contains the following keys:
 .. code-block:: json
 
    {
-    "git-hash": "0000000-dirty",
+    "git-desc": "v1.0-15-g42096-dirty",
+    "git-hash": "0000000",
```

Comment:
Can we make sure the desc and hash match up? I know this is an example but it'll be confusing otherwise.

Reply:
Good point, ill fix that right up. Consistant examples are much easier to read.